superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx

```diff
@@ -187,13 +188,10 @@ const FilterValue: React.FC<FilterControlProps> = ({
                   handleFilterLoadFinish();
                 })
                 .catch((error: Response) => {
-                  getClientErrorObject(error).then(parsedError => {
-                    errorRef.current = error.errors?.[0];
+                  getClientErrorObject(error).then(clientErrorObject => {
+                    setError(clientErrorObject);
+                    handleFilterLoadFinish();
                   });
-                  setError(
-                    error.message || error.error || t('Check configuration'),
```

Comment:
@michael-s-molina do we not need to preserve this messaging?

Reply:
Looking at `getClientErrorObject` implementation, and error is always returned and it deals with the generic case as well. In my opinion it offers a more standard treatment to the exceptions.